pkg/kubelet/cm/dra/state/state_checkpoint.go

```diff
@@ -54,11 +55,42 @@ type ClaimInfoState struct {
 	// PodUIDs is a set of pod UIDs that reference a resource
 	PodUIDs sets.Set[string]
 
+	// ResourceHandles is a list of opaque resource data for processing by a specific kubelet plugin
+	ResourceHandles []resourcev1alpha2.ResourceHandle
+
 	// CDIDevices is a map of DriverName --> CDI devices returned by the
 	// GRPC API call NodePrepareResource
 	CDIDevices map[string][]string
 }
 
+// ClaimInfoStateWithoutResourceHandles is an old implementation of the ClaimInfoState
```

Comment:
We also add the TODO to remove this with beta

Reply:
done